java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -1361,6 +1361,10 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_binaryOpVV(JNIEnv *env, j
         out->set_null_mask(std::move(new_mask), null_count);
       }
 
+      if (cudf::may_have_nonempty_nulls(*out)) {
```

Comment:
This is not needed here. Only fixed width types are supported for `out` as can be seen on line 1355, and fixed width types cannot have a nonempty null.